pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +309,37 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+func getLogWithStacktracing(logString string) (string, bool) {
+	reader := strings.NewReader(logString)
+	scanner := bufio.NewScanner(reader)
+	scanner.Split(bufio.ScanLines)
+
+	var log []string
+	var found bool
+	for scanner.Scan() {
+		line := scanner.Text()
+		// if it already found the stack we just want to keep add the log till the end
+		if found {
+			log = append(log, line)
+			continue
+		}
+		// tries to find the stack message at the given line and add it to the log
+		if found = stackTraceRegex.MatchString(line); found {
```

Comment:
Is the regex updated? I can't see it.

Reply:
Ops! I reverted it by mistake before committing it.